eZ/Publish/Core/Persistence/Legacy/User/Handler.php

```diff
@@ -78,7 +78,7 @@ public function __construct(Gateway $userGateway, RoleGateway $roleGateway, Mapp
      */
     public function create(User $user)
     {
-        $this->userGateway->createUser($user);
+        // Do nothing, creation is done via content handler.
```

Comment:
IMHO it should throw an exception.

Reply:
Or remove, it's anyway SPI layer

Reply:
In the end yes, but need to fine good way to handle user specific cache tags.

Reply:
Nothing. I was responding to removing whole method. 
This do nothing `\eZ\Publish\Core\Persistence\Legacy\User\Handler::create` and dont need to be called from this `\eZ\Publish\Core\Persistence\Cache\UserHandler::create` but the second method prevents us from deleting it from interface. (unless we get new way for tag handling).